app/views/shared/_user_photo.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if user.photo.present? %>
+    <div class="user-photo">
+      <%= image_tag user.photo, alt: user.name %>
+    </div>
+<% else %>
+    <div class="photo-placeholder">
+        No Photo uploaded
+    </div>
+<% end %>
```

Comment:
- [ ] Well done for using partials 👍👍 However, when I am using an image with the correct URL, this is how it is being displayed 👇 Kindly make sure to fix the width and height of the image according to the given wireframe. Also, please make sure to fix the image wherever it is used.

![image](https://github.com/misalazeem/Blog-App-Rails/assets/106119105/c859b23f-7f4a-47ce-b9cf-8376c0412a07)